symplyphysics/definitions/momentum_is_mass_times_velocity.py

```diff
@@ -0,0 +1,47 @@
+from symplyphysics import (
+    symbols, Eq, pretty, solve, Quantity, units,
+    validate_input, validate_output, expr_to_quantity
+)
+
+# Description
+## Momentum is the multiplication of velocity and mass
+## Definition: P = m * V
+## Where:
+## m is a mass of the object
+## V is it's velocity
+## P is momentum
```

Comment:
Every new definition and law should be covered with tests. See https://github.com/blackyblack/symplyphysics/blob/master/test/thermodynamics/pressure_from_temperature_and_volume_test.py for reference.